.env.example

```diff
@@ -13,7 +13,7 @@
 NODE_ENV=development
 PORT=3000
 # Websocket port
-NEXT_PUBLIC_WS_PORT=3001
```

Comment:
leaving a comment here for others who are wondering why its changed

**Reason**
- WSL usually uses 3001 (pointed out by @kflim)